ts-apps/decision-logic-visualizer/src/lib/layout-ir/paths-list.ts

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import {
```

Comment:
This is basically factored out from the old ladder-lir.svelte, as part of an effort to break up the old and too-big ladder-lir.svelte file into smaller pieces